### PR TITLE
Lightning beta

### PR DESCRIPTION
**Pull request by Banyutong**

### CODE REVIEW

File: src/data_interface/sages_pb2.py [hunk 1]
```diff
@@ -0,0 +1,815 @@
+# -*- coding: utf-8 -*-
```
Comment (guy-rosman-TRI) [inline src/data_interface/sages_pb2.py:1]: should be removed @Banyutong

File: README.md [hunk 2]
```diff
@@ -53,11 +57,11 @@ protoc --python_out=./ ./data_interface/sages.proto
 ```
 python ${SAIIL_PUBLIC}/src/data_interface/cholec_convert.py  ${SAIIL_PUBLIC}/data/annotations/cholec80_protobuf/ --phase-folder  ${SAIIL_PUBLIC}/data/cholec80/phase_annotations -v --tool-folder ${SAIIL_PUBLIC}/data/cholec80/tool_annotations
 ```
-
+* Split the protobuf annotation file in train and test subsets (train: video01 - video40, test: video41 - 80). Or you can found the split annotations in the [link](https://shorturl.at/BLOW7).
 ### Train an example phase classification network
 Run:
 ```
-${SAIIL_PUBLIC}/src/scripts/run_script_temporal_model.sh
+sh ./phase_net/train_model.sh
```
Comment (rosman9) [inline README.md:64]: use SAIIL_PUBLIC folder, or state assumed folder

File: README.md [hunk 3]
```diff
@@ -66,4 +70,21 @@ The code includes a phase classification network with:
 * LSTM temporal model.
 * Pytorch dataset/loader based on the protobufs defined in the SAGES 20' video/data annotation workshop in Houston.
 
-The main training script is under [src/scripts/train_temporal_model.py](scripts/train_temporal_model.py).
+The main training script is under [src/phase_net/train_baseline.py](src/phase_net/train_baseline.py).
```
Comment (rosman9) [inline README.md:73]: baseline is very generic, baseline for what? Are you saving the original for safety? If so, rename to "old_temporal" vs "temporal", etc

File: src/phase_net/train_baseline.py [hunk 4]
```diff
@@ -0,0 +1,258 @@
+import os
```
Comment (rosman9) [inline src/phase_net/train_baseline.py:1]: move from phase_net to temporal_models/phase_net

File: src/env.saiil.yml [hunk 5]
```diff
@@ -6,26 +6,26 @@ channels:
...
-  - pip=19.2.2=py36_0
+  - pillow
+  - pip
   - protobuf=3.9.1
   - pyexcel-ods3
-  - pytorch=1.3.0
-  - tensorboard=1.14.0
-  - tensorboardx=1.8
-  - tensorflow=1.14.0
+  - pytorch
```
Comment (rosman9) [inline src/env.saiil.yml:17]: why remove all of the version numbers? which ones did you remove on purpose vs. by accident?

File: src/phase_net/model.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+import pytorch_lightning as pl
+from torchvision import transforms
+
+# Model example for phase recognition training pipeline
+# MIT liscence
+# Author: Yutong Ban, Guy Rosman 
+
+
+
+class CNN_model(pl.LightningModule):
```
Comment (rosman9) [inline src/phase_net/model.py:13]: Does CNN_model have to be a pl.LightningModule? If you're only using the forward operator (true for any pytorch model), are you overconstraining use to use the network only with lightning at deployment time? Alternatively, is a lightning trainer enough?

**Pull request merged**